.github/workflows/pr_build.yml

```diff
@@ -182,20 +207,29 @@ jobs:
         with:
           rust-version: ${{env.RUST_VERSION}}
           jdk-version: ${{ matrix.java_version }}
-      - if: matrix.test-target == 'rust'
-        name: Rust test steps
-        uses: ./.github/actions/rust-test
-      - if: matrix.test-target == 'java'
-        name: Java test steps
+      - name: Java test steps
         uses: ./.github/actions/java-test
         with:
           maven_opts: -Pspark-${{ matrix.spark-version }},scala-${{ matrix.scala-version }}
 
+  macos-aarch64-test-rust:
+    name: macos-14(Silicon)/rust
```

Comment:
`fail-fast: false` is missing?

Reply:
`fail-fast` is for matrix builds, and there is no matrix for this job.